scripts/ai/controllers/ChopperController.lua

```diff
@@ -0,0 +1,60 @@
+---@class ChopperController : ChopperController
+

```

Comment:
This controller doesn't follow the convention of the other implement controllers, which are based on Spec's or external mods. Giants doesn't differentiate between combine or chopper, as both have a common combine controller for the common combine specialization. Unless you've got anything special which can be put here.

Reply:
I only did it to give clear separation between Chopper specific functions that CP team made to fix issues and the combine specific stuff. And as you said the combine stuff is related so I made sure to inherit the combine stuff just in case there was combine stuff used